html/common/templates/assets/assetsShopView.twig

```diff
@@ -257,6 +257,30 @@
             {% endfor %}
         </div>
     </div>
+    <div class="card-footer">
```

Comment:
Should this be card-header?

Reply:
Fixed in 6e5622aa

Reply:
Whoops I'm wrong - I thought this was at top of page sorry. Will fix :(